website/docs/network-traffic.mdx

```diff
@@ -91,28 +91,28 @@ Runs on the customer's own servers. We suggest <a href="https://configcat.com/su
 
 ### Delete the old feature flags and unused Targeting Rules
 
-If you have a lot of feature flags and Targeting Rules in a config, you can lower the size of the `config JSON` by deleting the old ones.
+If you have a large number of feature flags and Targeting Rules in a config, you can reduce the size of the `config JSON` by deleting the old ones. The [Zombie Flags](/zombie-flags) feature can help you finding stale feature flags.
```

Comment:
❤️

BTW, shouldn't we use relative urls in links within the docs?

Reply:
fixed